packages/csvviewer/src/toolbar.ts

```diff
@@ -28,12 +29,17 @@ export class CSVDelimiter extends Widget {
    * Construct a new csv table widget.
    */
   constructor(options: CSVToolbar.IOptions) {
-    super({ node: Private.createNode(options.selected, options.translator) });
+    super({
+      node: Private.createNode(options.widget.delimiter, options.translator)
+    });
+    this._widget = options.widget;
     this.addClass(CSV_DELIMITER_CLASS);
   }
 
   /**
    * A signal emitted when the delimiter selection has changed.
+   *
+   * @deprecated since 3.1
```

Comment:
Needs a bump to 3.2 or 4.0? Or maybe create another PR to include the deprecations (there are also deprecations of methods creating buttons) and rush it to 3.1?